<?xml version="1.0"?>
<launch>
    <!-- joy node -->
    <node respawn="true" pkg="joy" type="joy_node" name="ros_joy_node">
        <param name="dev" type="string" value="/dev/input/js0"/>
        <param name="deadzone" value="0.12"/> 
    </node>

    <!-- Axes 0.68 1.54 3 -->
    <param name="axis_linear" value="1" type="int"/>
    <param name="axis_angular" value="3" type="int"/>
    <param name="button_deadman_switch" value="2" type="int"/>

    <param name="scale_linear" value="1.8" type="double"/>
    <param name="scale_angular" value="2.54" type="double"/>
    <param name="twist_pub_topic" type="string" value="cmd_vel"/>

    <!-- JOY TO COMMAND VEL -->
    <node pkg="som_o_teleop" type="som_o_joystick" name="som_o_joystick" output="screen" />
</launch>
